### ISSUE TITLE
chapter-4/4-improving-accuracy-with-fine-tuning.ipynb does not contain the final finetuning section that is in book

### ISSUE THREAD

**Issue by gitgithan** [PracticalDL/Practical-Deep-Learning-Book#167]
There is a final finetuning throwing away dense and dropout layers that was expected according to the book but is missing in the notebook

**Comment by sidgan** [comment]
Thanks! Do you want to edit with a PR?
